diff --git a/authentik/api/v3/config.py b/authentik/api/v3/config.py
--- a/authentik/api/v3/config.py
+++ b/authentik/api/v3/config.py
@@ -71,6 +71,7 @@ class ConfigView(APIView):
         if (
             CONFIG.get("storage.media.backend", "file") == "s3"
             or Path(settings.STORAGES["default"]["OPTIONS"]["location"]).is_mount()
+            or CONFIG.get_bool("media.enable_upload")
             or deb_test
         ):
             caps.append(Capabilities.CAN_SAVE_MEDIA)
