diff --git a/lifecycle/system_migrations/tenant_files.py b/lifecycle/system_migrations/tenant_files.py
index 40795d460..7ac1efb34 100644
--- a/lifecycle/system_migrations/tenant_files.py
+++ b/lifecycle/system_migrations/tenant_files.py
@@ -2,8 +2,9 @@
 from pathlib import Path
 
 from lifecycle.migrate import BaseMigration
+from authentik.lib.config import CONFIG
 
-MEDIA_ROOT = Path(__file__).parent.parent.parent / "media"
+MEDIA_ROOT = Path(CONFIG.get("storage.media.file.path"))
 TENANT_MEDIA_ROOT = MEDIA_ROOT / "public"
 
 
